test.js

```diff
@@ -202,6 +203,13 @@ describe('chalk.level', () => {
 		assert.equal(chalk.level, 1);
 		chalk.level = oldLevel;
 	});
+
+	it('should disable colors if they are not supported', () => {
+		return execa('node', ['fixture'])
+			.then(result => {
+				assert.equal(result.stdout, 'test');
+			});
+	});
```

Comment:
Yes, I think so, because it fails everywhere: on CI and on my laptop. It fails when publishing a package with `np`, spawning a process manually in repl, in tests. But if you run `node fixture` directly, it works as expected. I was able to reproduce the crash every time, so I think this failing test is stable.